src/System.Numerics.Tensors/pkg/System.Numerics.Tensors.pkgproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <!-- we need to be supported on pre-nuget-3 platforms (Dev12, Dev11, etc) -->
     <MinClientVersion>2.8.6</MinClientVersion>
+    <IsShipping>false</IsShipping>
```

Comment:
@wtgodbe @tannergooding for future packages that are experimental and don't follow the name convention, like `Private`, `Experimental` in the name, with this changes we can now just simply add this propety to the `pkgproj` and will do the right thing and mark it as non-shipping.

Reply:
Awesome!